### PR TITLE
Add minor changes in AlloyNav

### PR DESCRIPTION
**Pull request by sarajul430**

### CODE REVIEW

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+                <ul>
+                    <a>Try it free</a> <select></select>
+                </ul>
+
+            </div>
+        </nav>
+       </section>
+        
+
```
Comment (amezng) [inline marketing/src/sections/navigation/AlloyNav/AlloyNav.jsx:44]: Remove extra spaces.

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+nav{
```
Comment (amezng) [inline marketing/src/sections/navigation/AlloyNav/AlloyNav.css:1]: add classnames instead of using generic tag for creating css rules.

**Review by amezng — CHANGES_REQUESTED** [review]

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.jsx (hunk 1 above)
Comment (sarajul430) [reply marketing/src/sections/navigation/AlloyNav/AlloyNav.jsx]: I have remove extra spaces

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css (hunk 2 above)
Comment (sarajul430) [reply marketing/src/sections/navigation/AlloyNav/AlloyNav.css]: I have add className

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css [hunk 3]
```diff
@@ -9,11 +9,11 @@ select{
     border: none;
 }
 
-nav > div:first-child{
+.navbar > div:first-child{
```
Comment (amezng) [inline marketing/src/sections/navigation/AlloyNav/AlloyNav.css:12]: Preferably use allow-nav

**Review by amezng — CHANGES_REQUESTED** [review]

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css (hunk 3 above)
Comment (sarajul430) [reply marketing/src/sections/navigation/AlloyNav/AlloyNav.css]: i have added className alloy-nav

**Review by amezng — CHANGES_REQUESTED** [review]

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css [hunk 4]
```diff
@@ -70,13 +71,13 @@ ul {
     align-items: center;
     position: absolute;
     border: 1px solid;
-    margin: 45px -145px;
+    margin: 49px -170px;
```
Comment (amezng) [inline marketing/src/sections/navigation/AlloyNav/AlloyNav.css:74]: Use display flex

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css [hunk 5]
```diff
@@ -70,13 +71,13 @@ ul {
...
     padding: 0px 50px;
     border-radius: 8px;
     background-color: white;
     border: none;
     box-sizing: border-box;
-    width: 1055px;
+    width: 1160px;
```
Comment (amezng) [inline marketing/src/sections/navigation/AlloyNav/AlloyNav.css:80]: use max-width as 1200 px

**Review by amezng — CHANGES_REQUESTED** [review]

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css (hunk 5 above)
Comment (sarajul430) [reply marketing/src/sections/navigation/AlloyNav/AlloyNav.css:79]: I have  set width

**Review by amezng — APPROVED** [review]

**Pull request merged**